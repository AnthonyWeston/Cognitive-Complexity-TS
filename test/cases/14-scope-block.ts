{
    // depth 0
    while (true); // +1
}