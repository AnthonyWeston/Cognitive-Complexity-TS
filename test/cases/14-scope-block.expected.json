{
    "14-scope-block.ts": {
        "score": 1
    }
}